notebooks/Data_visualization/Data_visualizations.ipynb

```diff
@@ -0,0 +1,474 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    bike = pd.read_csv("Bike.csv")
</pre><p>it might be worth doing a bike.head() here to show the dataset we're working with here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/data-science-workflows/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>